src/feature_builder.hpp

```diff
@@ -167,69 +174,73 @@ struct overzoomed_feature_builder
         vtzero::decode_linestring_geometry(feature.geometry(), detail::line_string_handler<coordinate_type>(multi_line, dx_, dy_, zoom_factor_));
         std::vector<mapbox::geometry::line_string<coordinate_type>> result;
         boost::geometry::intersection(multi_line, bbox_, result);
-        bool valid = false;
-        vtzero::linestring_feature_builder feature_builder{layer_builder_};
-        feature_builder.copy_id(feature);
-
-        for (auto const& l : result)
+        if (!result.empty())
         {
-            if (l.size() > 1)
+            bool valid = false;
+            vtzero::linestring_feature_builder feature_builder{layer_builder_};
+            feature_builder.copy_id(feature);
+            for (auto const& l : result)
```

Comment:
With variable names like `result` and `l` it is difficult to follow the code here.

Reply:
Agree. However, I will like @millzpaugh @artemp handle this kind of cleanup in `master` after this merges.